bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3731,17 +3731,17 @@ spec:
   relatedImages:
   - image: docker.io/dellemc/dell-csm-operator:v1.5.1
     name: dell-csm-operator
-  - image: docker.io/dellemc/csi-isilon:v2.9.1
+  - image: docker.io/dellemc/csi-isilon:v2.10.1
     name: csi-isilon
-  - image: docker.io/dellemc/csi-powermax:v2.9.1
+  - image: docker.io/dellemc/csi-powermax:v2.10.1
     name: csi-powermax
   - image: docker.io/dellemc/csipowermax-reverseproxy:v2.8.1
     name: csipowermax-reverseproxy
```

Comment:
@chimanjain Can you please confirm if there is no update for reverseproxy in patch release?

Reply:
@rajendraindukuri I have made the updates from driver side. Should we update the modules in the same PR.